source/develop/release-management/features/gluster/gluster-dr.html.md

```diff
@@ -0,0 +1,125 @@
+* Slave volume needs to have similar capacity as the master volume
+
+![Gluster-Geo-replication](gluster-dr-georep.png) 
+
+Once the geo-replication session is configured, following steps can be scripted to ensure periodic sync
+
+1. Establish connection to running oVirt engine 
+2. Query for list of running VMs
+3. Create snapshot of all running VMs to ensure that data is quiesced and in a consistent state at the secondary site
+4. Execute the georep_scheduler script that:
+    - creates a checkpoint
+    - starts geo-replication, copying files from master volume (storage domain) to the slave volume located at central backup site
+    - Stops the geo-replication session once checkpoint is completed
+5. Mount the slave volume from secondary site and delete the external snapshot image file (overlay image) that was created as part of script
+6. At oVirt engine, delete the created snapshots via API that will call live merge of image
+7. On completion, post an event to oVirt. The event is an Alert with error message in case the process did not complete successfully. Successful completions are noted as normal events. Administrator can subscribe to external events to receive the events via email.
+
+Such a script is already available at [ovirt-georep-backup](https://github.com/sabose/ovirt-georep-backup)
+
+### Recovery on disaster
+
+In the event of a disaster, the storage domain can be attached to a running instance of oVirt. 
+
+1. A new instance of oVirt is setup with a master storage domain.(see [Quick_Start_Guide#Install_oVirt](/Quick_Start_Guide#Install_oVirt)). A master storage domain needs to be active in oVirt to initialize the Data Center and perform further operations
+2. Use the [Import Storage Domain](/Features/ImportStorageDomain) feature to import the gluster volume from secondary site (the one setup as slave gluster volume in previous setup)
+    - In case the storage domain contains the VMs and all its disks, the VMs can be imported to the new oVirt instance
+    - In case the storage domain contains only floating disks (i.e not attached to any VMs or where the storage domain does not contain the VM's OS disks), the disks can be registered via GUI (see [Bug 1138139](https://bugzilla.redhat.com/show_bug.cgi?id=1138139))
+        * There's currently an issue where disks with snapshots cannot be registered. The overlay image file needs to be manually deleted from the storage domain before this is possible.
+
+
+To integrate this solution better into oVirt and provide users a seamless way of managing and monitoring the DR solution, we need enhancements to this approach. Sections below will outline the design to do so.
+
+## Enhancements
+
+### UX
+
+* Allow enabling sync for disaster recovery on storage domain. This will currently be available for gluster storage domains alone, but can be extended to cover other storage domain types later
```

Comment:
How about a right-click on the storage domain that opens the menu for it with a 'sync now' button?